wikimedia/wikipedia-ios
# PR feedback for auto article description summary

**Phabricator:**
https://phabricator.wikimedia.org/T351649

### Notes
This addresses my PR feedback for #4745 to get translations working. 

### Test Steps
Test wiki doesn't give the option to add an article description, I think due to PCS-side issues, but we can test description updates.

1. Go to an article with a description on Test Wiki. Tap first section pencil, you should see a prompt to edit the article description.
2. Change article description, then publish. After article refreshes, go to article history. Confirm you see a minor edit summary with "Updated short description" as the text. You should also see an edit summary tag appended `#article-change-desc`.
